### DISCUSSION TITLE
What kid of game should we make?

### DISCUSSION THREAD

**Discussion by jeffaroundtheworld** [jeffaroundtheworld/2024gamecollab#6] [Announcements]
I have no idea... I just run the show.  Thought from the team?

**Comment by kodriq** [comment]
can it have a plot

**Comment by Jackcarlisle1** [comment]
souls-like?

**Comment by Rhyswatsonn** [comment]
maybe a first person game where you roam around with a goal. a cool feature could be a flashlight you carry like silent hill? to add some creepy ambiance

**Reply by ajleach9689** [reply]
set in the school?
